### DISCUSSION TITLE
Speak Readings and alarms not work anymore after canceling an xDrip alarm from my garmin watch

### DISCUSSION THREAD

**Discussion by FriedelPas** [NightscoutFoundation/xDrip#2724] [Q&A] [answered]
Speak readings and alarms and sound alarms are one of the essential aspects of the xDrip app for me. It make it much easier to stay most of the time in range with your glucose values. Especialy while doing sport.

Since this evening this is not working anymore. I believe that is after I got the sound and alarm of to high sugar. I got also a message of that on my garmin Forerunner 255 smartwatch. There I selected for the first time to cancel the alarm. After that my phone is not speaking anymore the readings and I have also not hear anymore the alarms.
When I go in the alarms and alerst and test the alert, that works. When I disable and enable the speaking readings, he is not speaking the reading. I tried several times to startup again the phone but that was not working.

My phone is a Samsung Galacy S21Fe with Android 12
My xDrip version is 344deb4-2022.11.16.
I uploaded via xDrip the logbook.

How can I solve that isue?

**Comment by FriedelPas** [comment]
But when I connect an bleutooth ear, in that ear it works fine. When not connected anymore it works not anymore.

**Comment by Navid200** [comment]
Are you sure you haven't accidentally lowered the phone volume?  
The speak reading volume is controlled by the media volume on your phone.

**Reply by FriedelPas** [reply]
I have checked that several times and all vollumes are on maximum.

**Comment by FriedelPas** [comment, marked as answer]
It was something with the bleutooth connection. After a couple of times connecting and disconnecting the bleutooth ears it works again!
